docs/samples/hub-state-import-data-sample.json

```diff
@@ -0,0 +1,13 @@
+[{"id": "my_config", "schemaVersion": "2.0", "labels": null, "content": {"deviceContent": {"properties.desired.awesome": 3}}, "targetCondition": "tags.environment='test'", "createdTimeUtc": "2022-06-28T22:56:32.937713Z", "lastUpdatedTimeUtc": "2022-06-28T22:56:32.937713Z", "priority": 10, "systemMetrics": {"results": {"targetedCount": 0, "appliedCount": 0}, "queries": {"targetedCount": "select deviceId from devices where tags.environment='test'", "appliedCount": "select deviceId from devices where configurations.[[my_config]].status = 'Applied'"}}, "metrics": {"results": {}, "queries": {}}, "etag": "MQ=="}, {"id": "my_config2", "schemaVersion": "2.0", "labels": null, "content": {"deviceContent": {"properties.desired.prop2": 5}}, "targetCondition": "*", "createdTimeUtc": "2022-06-28T23:03:31.8831286Z", "lastUpdatedTimeUtc": "2022-06-28T23:03:31.8831286Z", "priority": 7, "systemMetrics": {"results": {"targetedCount": 3, "appliedCount": 3}, "queries": {"targetedCount": "select deviceId from devices", "appliedCount": "select deviceId from devices where configurations.[[my_config2]].status = 'Applied'"}}, "metrics": {"results": {}, "queries": {}}, "etag": "MQ=="}]
```

Comment:
I'm not a fan of this data structure for holding hub state. Ideally it should be an object with aptly named keys holding the various aspects of state.

i.e.
```
{
  "devices": [{"identity": {}, "twin": {}}],
  "modules": [{"identity": {}, "twin": {}}],
  "configurations": [{}, {}]
  ...
  ...
}
```

That is just an example, but you can see with hub state starting with a root level object, it is easier to reason through its attributes and actually use/iterate through it.